openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -96,7 +96,6 @@ def _create_args(details, id_):
     args = {
         "foreign_landing_url": details[1],
         "image_url": details[2],
-        "thumbnail_url": details[3],
         "license_info": get_license_info(license_url=details[6]),
```

Comment:
Thank you for catching this, @stacimc ! I've refactored phylopic script to use a dict instead of a list to make it easier to understand what data is coming from where.